src/libs/DateUtils.js

```diff
@@ -175,6 +175,17 @@ function getDBTime(timestamp = '') {
         .replace('Z', '');
 }
 
+/**
+ * Returns the datetime string one millisecond before a given datetime string
+ *
+ * @param {String} dateTimeString in the format expected by the database
```

Comment:
The challenging part is that we don't use a standard format which has a name (I don't think). Our format is this `2022-12-30 20:49:11.234`. ISO format would be `2022-12-30T20:49:11.234Z`. I think that's why we've been describing it as DB time. Because that's the format our DB uses.

Reply:
Oh hmm.. maybe we can add to the comment what the format is as a reminder? I think my concern is that people will see this and ask, "what format is expected by the database?"

Reply:
Actually, I think we can just leave that detail out since it can in theory accept any format that `moment` supports. Updated.

Reply:
Actually, I think we can just leave that detail out since it can in theory accept any format that `moment` supports. Updated.